pipelines/components/image_classifier_keras/src/hdf5.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2018 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""
+"""
+
+import h5py
+import numpy as np
+
+def store_dataset(name, dataset, verbose=False):
+    """ Store a dataset of preprocessed image data to HDF5 file, where each HDF5 dataset is a collection of preprocessed
```

Comment:
Gonzalo and I noticed the system complains when we use 4-spaces for indentation (which I find it strange, since 4-space is pretty standard anyway). Just a heads-up that you will likely to face the same issue as well.

Reply:
OK. I will keep the ident spacing issue in mind.